code/modules/clothing/under/color.dm

```diff
@@ -30,6 +33,10 @@
 	item_state = "b_suit"
 	_color = "blue"
 	flags = ONESIZEFITSALL
+	species_fit = list("Vox")

```

Comment:
There's a much easier way to do this then, with far less code.

species_fit and sprite_sheets for /obj/item/clothing/under/color

All the others will inherit this automatically (or should).